src/libstore/unix/build/external-derivation-builder.cc

```diff
@@ -106,6 +106,9 @@ struct ExternalDerivationBuilder : DerivationBuilderImpl
 
                 args.insert(args.end(), jsonFile);
 
+                if (chdir(tmpDir.c_str()) == -1)
```

Comment:
This isn't really necessary anymore, but I left it because it's probably the right thing to do...? idk, happy to drop